topics/admin/tutorials/ansible/tutorial.md

```diff
@@ -295,7 +295,7 @@ The above introduction was certainly not enough for you to feel confident in Ans
 >       > ```
 >       {: .solution }
 >
->       > <details-title>For your own infrastructure, do you connect with a different user? Password? SSH key?</details-title>
+>       > <details-title>For your own infra, do you connect as a different user? Password? SSH key?</details-title>
```

Comment:
```suggestion
>       > <details-title>For your own infrastructure, do you connect as a different user? Password? SSH key?</details-title>
```

we need to keep the longer form of the word here, it should be more accessible for non-native english speakers, as well as those reading the materials through machine translation where infrastructure is more likely to be translated correctly than the informal slang term 'infra' that we all use :)